src/components/Login.jsx

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react'
+
+export default class Login extends Component {
+    render() {
+        return (
+            <form onSubmit={this.handleSubmit} method="post">
```

Comment:
Cover form with div

Reply:
The form itself encloses all the content. Is it still necessary?